### DISCUSSION TITLE
Add support for URL icons

### DISCUSSION THREAD

**Discussion by gripfastistech** [upptime/upptime#105] [Ideas] [answered]
add icon images before site titles in static site: _Originally posted by @gripfastistech in https://github.com/upptime/upptime/issues/70#issuecomment-737584539_

I think it's just easier to add an icon feature for each site rather than trying to add an `<img...` in the YAML site title. Can you open a separate issue for "Add support for URL icons"? Then we can show these icons in the README and on the status website.

_Originally posted by @AnandChowdhary in https://github.com/upptime/upptime/issues/70#issuecomment-737702822_

**Comment by AnandChowdhary** [comment, marked as answer]
Custom icon support has been added to the README.md file, and will soon be added to the status page website as well. You can add a custom icons like this:

```yaml
sites:
  name: ...
  url: ...
  icon: https://example.com/favicon.png
```

If you don't provide an icon, we'll show you the icon using GitHub's favicon service with a fallback to a globe icon:

![Screen Shot 2020-12-06 at 13 13 39](https://user-images.githubusercontent.com/2841780/101274538-e3ce5500-37c4-11eb-932a-4e81137d68c9.png)

**Comment by AnandChowdhary** [comment]
Icons are also available on the status site now:

![Screen Shot 2020-12-06 at 14 47 09](https://user-images.githubusercontent.com/2841780/101276305-f7cc8380-37d1-11eb-8ada-88d69832120c.png)
